pkg/vdri/peer/did.go

```diff
@@ -55,9 +61,10 @@ func NewDoc(publicKey []did.PublicKey, authentication []did.VerificationMethod,
 	return doc, nil
 }
 
-// computeDid creates the peer DID.
+// computeDidMethod1 creates the peer DID.
```

Comment:
what does the 1 suffix signify?

Reply:
The numeric algorithm used to compute the method-specific id:

> In this version of the spec, the multicodec descriptor for numalgo == 0 is dependent on key type and exactly matches the descriptor that would be used for the same value in did:key. The multicodec descriptor for numalgo == 1

[Source](https://identity.foundation/peer-did-method-spec/#method-specific-identifier)